pages/common/tesseract.md

```diff
@@ -0,0 +1,23 @@
+# Tesseract
+
+> OCR (Optical Character Recognition) engine.
+
+- Recognize text in image `image.png` and save to `content.txt` file:
```

Comment:
Its better to rename the file as output.txt, and put the same name in the example too. Makes it uniform with other pages that we have.